.github/workflows/nix.yaml

```
@@ -111,28 +108,60 @@ jobs:
+      - name: Install crane
+        run: |
+          nix profile install nixpkgs#crane
+          export PATH="$HOME/.nix-profile/bin:$PATH"
+
+      - name: Create and push manifest
+        run: |
+          SHORT_SHA=$(git rev-parse --short HEAD)
+          MANIFEST_TAG="ghcr.io/${{ github.repository }}:${SHORT_SHA}"
+          AMD64_IMAGE="${MANIFEST_TAG}-amd64"
+          ARM64_IMAGE="${MANIFEST_TAG}-arm64"
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)